server/include/ai/team.h

```diff
@@ -0,0 +1,38 @@
+/*
+** EPITECH PROJECT, 2024
+** Zappy Server
+** File description:
+** Team struct
+*/
+
+#pragma once
+
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#include "list/list.h"
+
+typedef struct s_team {
+    list_t *list_ai;
+    char *name;
+    size_t max_place;
+} team_t;
+
+/**
+ * @brief Create a team object.
+ *
+ * @param name Name of the team.
+ * @param max_place Max place in a team
```

Comment:
Don't forget the point.


```suggestion
 * @param max_place Max place in a team.
```